kite-data/kite-data-hbase/src/main/java/org/kitesdk/data/hbase/avro/AvroEntitySerDe.java

```diff
@@ -331,20 +332,32 @@ private void initKACRecordDatumMaps(String fieldName, Schema fieldSchema,
     }
   }
 
+  private static Schema unwrapNull(Schema schema) {
```

Comment:
Should this be named `unwrapNullable`?

Is it okay to assume that this is never called on a union other than of some type with null?